.idea/checkstyle-idea.xml

```diff
@@ -10,7 +10,7 @@
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
         <entry key="location-2" value="PROJECT_RELATIVE:$PROJECT_DIR$/config/metrics/checkstyle/checkstyle.xml:Terasology" />
         <entry key="property-2.samedir" value="config/metrics/checkstyle" />
-        <entry key="scan-before-checkin" value="false" />
+        <entry key="scan-before-checkin" value="true" />
```

Comment:
What kind of scans does this include? 

@keturn to the rescue? :upside_down_face:

Reply:
Honestly, I didn't notice this file is added into the commit stage. 😅 
But, I can assume it happened because I enabled the Checkstyle scan before VCS check-in (aka before commit).

Reply:
Yes please - while that's possibly a good practice it is probably better if we leave it up to individual contributors for now, not all of whom may even have the Checkstyle plugin installed (I don't remember if that's a default one) :-)

Probably would be a different project to actually get the Checkstyle config and docs to the point where we could consider doing this for real. We'd want to make sure we have good docs on what a false positive is, for instance, or that might confuse contributors and lead to unrelated "fixes" in PRs we then have to ask to have undone 😅

Reply:
Then we should find a way to ignore these change by default (put `.idea` into the git-ignored list?). I think if a user changes this setting, it will automatically reflect in a change in this checked-in config file.... =/

Reply:
This looks like a topic for another venue. For the purposes of this PR, if it's not obviously the right setting, go ahead and revert the change to this file.

Reply:
Alright I will revert the change and try to avoid adding those unrelated files by accident 😅

Reply:
Alright I will revert the change and try to avoid adding those unrelated files by accident 😅